code/modules/vore/fluffstuff/custom_guns_vr.dm

```diff
@@ -891,7 +677,7 @@
 	mag_type = MAGAZINE
 
 /obj/item/ammo_casing/stg
-	desc = "A 7.92×33mm Kurz casing."
+	desc = "A 7.92 33mm Kurz casing."
 	icon_state = "rifle-casing"
 	caliber = "kurz"
 	projectile_type = /obj/item/projectile/bullet/rifle/a762
```

Comment:
I removed the strange non-unicode X character. I have no idea what character that is, but it looks insane and can't be rendered right ingame. Just removed it while I was there. Can readd a normal X or something later.